File: src/lib/tests/res/mixed/Source.java
public class Source {

    public static void main(String[] args) {
        System.out.println("Some text");
        boolean b = false;
        int i = 42;
        switch(i) {
            case 0:
                break;
            case 1:
                b = true;
                break;
            default:
                ++i;
                break;
        }
        System.out.println(i);
        System.out.println(String.valueOf(b));
    }

}

File: src/lib/tests/res/mixed/source.c
#include <stdio.h>
#include <stdbool.h>

int main(void) {
    printf("Some text\n");
    bool b = false;
    int i = 42;
    switch(i) {
        case 0:
            break;
        case 1:
            b = true;
            break;
        default:
            ++i;
            break;
    }
    printf("%d\n", i);
    printf("%s\n", b ? "true" : "false");
    return 0;
}
